clang/test/SemaSYCL/check-indirect-attribute-propagation.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -std=c++11 -fsyntax-only -fsycl -fsycl-is-device -verify %s
```

Comment:
I think this test is no longer needed. Seems to be a duplicate "check-notdirect-attribute-propagation.cpp" expecting opposite behavior from warning :)

Reply:
I will remove the test check-indirect-attribute-propagation.cpp and keep check-notdirect-attribute-propagation.cpp. There is still a need to show that transitive calls's attributes are propagated, isn't there?

Reply:
Yes. I think adding a codegen test is also helpful. I assume there are existing tests for testing attribute propagation from 'direct' function calls. Maybe add an indirect call there as well.

Reply:
Ok. Will do.

Reply:
Still unresolved